reservation-service/src/main/java/com/revature/dataLooader/BootstrapData.java

```diff
@@ -33,15 +34,15 @@ public void run(String... args) throws Exception {
 
         Reservation reservation1 = new Reservation(
                 9,1,2,1,1, RoomType.PHYSICAL,"Revature CEO",
-                "01-17-2021 09:00","01-17-2021 17:00");
+                "01-17-2021 09:00","01-17-2021 17:00", RoomOccupation.MEETING);
         
         Reservation reservation2 = new Reservation(
                 6,1,1,1,2,RoomType.PHYSICAL,"Revature CEO",
-                "01-18-2021 09:00","01-18-2021 17:00");
+                "01-18-2021 09:00","01-18-2021 17:00", RoomOccupation.TRAINING);
         
         Reservation reservation3 = new Reservation(
                 8,1,2,1,2,RoomType.VIRTUAL,"Revature CEO",
-                "01-19-2021 09:00","01-19-2021 17:00");
+                "01-19-2021 09:00","01-19-2021 17:00", RoomOccupation.MEETING);
         
         Reservation reservation4 = new Reservation(
                 7,1,1,1,RoomType.PHYSICAL,"Revature CEO",
```

Comment:
Is room occupation a required variable?

Reply:
yes, since the location team change the design. Savita told me now physical is not for meeting room. RoomOccupation.MEETING makes the room is meeting room. I am still waiting the answer from Location team for double check. please do not merge this.